src/form/Input.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/HTMLWidget", "css!./Input"], factory);
+    } else {
+        root.form_Form = factory(root.common_HTMLWidget);
+    }
+}(this, function (d3,HTMLWidget) {
+    function Input() {
+        HTMLWidget.call(this);
+        this._class = "form_Input";
+
+        this._tag = "div";
+    }
+    Input.prototype = Object.create(HTMLWidget.prototype);
+    Input.prototype.publish("id", "", "string", "HTML id for the input");
```

Comment:
This clashes with Widget.id() and Widget._id - will need to be renamed (then my comment above will be invalid).